Makefile

```diff
@@ -48,7 +48,11 @@ api:
 		--go-grpc_out=paths=source_relative:./api \
 		--validate_out=paths=source_relative,lang=go:./api \
 		--openapi_out=fq_schema_naming=true,default_response=false:. \
-		$(API_PROTO_FILES)"
+		$(API_PROTO_FILES) && \
+		cd api/ && \
+		buf lint && \
```

Comment:
Another option is that you could add the `protoc-gen-buf-lint` (https://buf.build/docs/reference/protoc-plugins) and `protoc-gen-buf-breaking` protoc plugin to the protoc command line, above, and possible avoid executing buf.

Or alternatively, and maybe better, we could avoid executing the protoc command and just do `buf generate` before lint and breaking as part of `make api`. I don't want to delay the PR, just pointing out that both tools should do the same thing in different ways and we probably only need one.

Reply:
I appreciate all the feedback so no worries delaying. I agree that only having a single tool like buf would be ideal, I'll see how I can update the make targets to use buf instead, should be very similar

Reply:
Okay so after having a go with buf, I got buf generate working and should be 1-1 with how we had protoc setup with lint & breaking support now. I am still using the local versions of most of the protoc plugins in our docker image except for one (github.com/google/gnostic/cmd/protoc-gen-openapi) which was having an odd issue with how openapi.yaml was being generated with `buf generate`. Using its remote hosted version instead @ https://buf.build/community/google-gnostic-openapi worked just fine. 

No more long protoc commands, just a long buf.gen.yaml file yay!